### DISCUSSION TITLE
post/hackergame-2023-hole

### DISCUSSION THREAD

**Discussion by giscus** [GrassBlock1/GrassBlock1#6] [Blog]
# post/hackergame-2023-hole

【本文章使用Notion导出】参黑客原神，品折腾人生（？）

https://lab.imgb.space/post/hackergame-2023-hole

**Comment by wuhang2003** [comment]
hg 病友兼故事另一位主人公前来划水（
1.搜索的话，知道一个搜不出来那肯定得换其他的，人不能吊死在一棵树上（
2.截图那题我试了一下，即使不知道它可能是个漏洞，单搜 pixel 截图也能在后两三页翻到对应的新闻，实际就看你能不能想到这是个漏洞利用题了
3.git 这种东西，不会的高级操作还是跟着搜索引擎指引用命令行吧
4.图片不一样正常，每个人的 flag，甚至包括旅行照片给的图都是不一样的
5.之前聊天也说过 hg 不算是纯 ctf，（前面的题）更多的是考信息收集与学习运用的能力

**Reply by GrassBlock1** [reply]
新病友来回复了（什么
1. ~~终于明白为什么浏览器要给我十个甚至九个搜索引擎选择了~~
2. 没想到，我以为是什么机型代号+加密（？
3. 只能说git命令行是万物归宗了（
4. 这样子！~~第一年来参加比赛没经验，明年看着这个应该能避开一些坑~~
5. ~~我先把不纯的ctf玩明白再说~~，应该多多学习了...
